front-end/.gitignore

```diff
@@ -37,6 +37,8 @@ npm-debug.log
 yarn-error.log
 testem.log
 /typings
+src/environments/environment.ts
+dist/
```

Comment:
There's a reason this isn't git ignored. I don't have a CI process to build the front-end, so I just build it locally to speed up deployments

Reply:
Do you want me to help you make a quick and easy CI?
It wont be a master-class one, but it'll work.